control_panel_api/tests/test_models.py

```diff
@@ -0,0 +1,79 @@
+from django.test import TestCase
+
+from control_panel_api.models import (
+    Role,
+    Team,
+    TeamMembership,
+    User,
+)
+
+
+class FixtureTestCase(TestCase):
```

Comment:
These tests should all be methods on a single class, rather than subclasses of `FixtureTestCase`.

Reply:
Alright, moving everything in a `TestCase` as these tests are all related to the membership 👍